pci.cpp

```diff
@@ -410,6 +410,23 @@ PciHandle::~PciHandle()
 
 #ifndef PCM_USE_PCI_MM_LINUX
 
+int PciHandle::openMcfgTable() {
+    const char* path = "/sys/firmware/acpi/tables/MCFG";
+    int handle = ::open(path, O_RDONLY);
+
+    if ( handle < 0 ) {
+        /**
+         * There are no MCFG table on some machines, but MCFG1.
+         * See https://github.com/opcm/pcm/issues/74 for details
+         */
+        path = "/sys/firmware/acpi/tables/MCFG1";
+        handle = ::open(path, O_RDONLY);
+        std::cerr << "Can't open MCFG table. Check permission of /sys/firmware/acpi/tables/MCFG or MCFG1" << std::endl;
```

Comment:
it will report error also if "/sys/firmware/acpi/tables/MCFG1" file has been successfully opened.

Reply:
Shit! My bad. :)